src/main/java/practice/StreamPractice.java

```diff
@@ -35,7 +88,11 @@ public Double getOddNumsAverage(List<Integer> numbers) {
      * Example: select men who can be recruited to army (from 18 to 27 years old inclusively).
      */
     public List<Person> selectMenByAge(List<Person> peopleList, int fromAge, int toAge) {
-        return Collections.emptyList();
+        return peopleList.stream()
+                .filter(person -> person.getSex() == Person.Sex.MAN)
+                .filter(person -> person.getAge() >= fromAge)
+                .filter(person -> person.getAge() <= toAge)
+                .collect(Collectors.toList());
```

Comment:
```suggestion
                .toList();
```
fix the same issue in all places

Reply:
\+